tokens/PRIZE

```diff
@@ -1 +0,0 @@
-
```

Comment:
{
    "tokenId": "tti_e1e29fe12d91a7f63ce273de",
    "overview":{
      "en":"A bull running with the power of the community. community piggy bank. Fastest and safest payment tool available for future trade."
    },
    "links":{
      "website":["https://www.prizefinance.io"],
      "github":["https://github.com/PrizeFinance/Prize"],
      "telegram":["https://t.me/prizefinancecommunity"],
      "twitter":["https://twitter.com/prizefinanceio"],
      "instagram":["www.instagram.com/prizefinance.io"],
      "explorer":["https://viteview.xyz/#/token/tti_e1e29fe12d91a7f63ce273de"]
    }
  }